torch_xla/csrc/cross_replica_reduces.cpp

```diff
@@ -275,38 +275,59 @@ RecvResult BuildRecvWithToken(xla::XlaOp token, const xla::Shape& recv_shape,
 }
 
 ReduceScatterResult BuildReduceScatter(
-    AllReduceType reduce_type, xla::XlaOp input, xla::XlaOp token, double scale,
-    int64_t scatter_dim, int64_t shard_count,
+    AllReduceType reduce_type, absl::Span<const xla::XlaOp> inputs,
```

Comment:
can we do the similar thing here? not changing the default behavior of the reducescatter?

Reply:
fixed